### PR TITLE
Feature/issue #70

### PR DESCRIPTION
**Pull request by amba178**
AP<>JS join  table

### CODE REVIEW

**Comment by joaopapereira** [comment]
We are missing here the agency_people  table

File: db/migrate/20151026060621_agency_people_job_seekers_join_table.rb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+class AgencyPeopleJobSeekersJoinTable < ActiveRecord::Migration
+  	def change
+    	create_table :agencies_seekers, id: false do |t|
```
Comment (joaopapereira) [inline db/migrate/20151026060621_agency_people_job_seekers_join_table.rb]: The table should be renamed to seekers_agency_people.
And also the file

**Comment by joaopapereira** [comment]
Please do the changes in the comments and push again

**Comment by amba178** [comment]
creat new PR

**Pull request closed without merging**